framework-docs/modules/ROOT/pages/web/webmvc/mvc-ann-async.adoc

```diff
@@ -430,6 +430,12 @@ returns a xref:web/webmvc/mvc-ann-async.adoc#mvc-ann-async-reactive-types[reacti
 are written to the Reactor `Context` as key-value pairs, using the key assigned by the
 `ThreadLocalAccessor`.
 
+In addition, for `ThreadLocal` managed values `LocaleContext` and `RequestAttributes`
+(accessed by `LocaleContextHolder` and `RequestContextHolder` respectively), users have
+the option to opt-in to context propagation through the `ThreadLocalAccessor` implementations,
+`LocaleContextThreadLocalAccessor` and `RequestAttributesThreadLocalAccessor`.
+(They are not auto-registered by default.)
+
```

Comment:
I think these implementations can be mentioned at the end of this section, after the scenarios are discussed. Something along the lines of..


The following `ThreadLocalAccessor` implementations are provided out of the box:

- `LocaleContextThreadLocalAccessor` -- propagates `LocaleContext` via `LocalContetHolder.
- ...

The above are not registered automatically. You will need to register them with the `ContextRegistry.getInstance()` on startup.

Reply:
Thanks for this change. However, I still think this whole addition needs to move to end of the section. Otherwise it breaks up the flow, and the next paragraph "For other asynchronous handling scenarios..." is now referring to content that's further up.